### ISSUE TITLE
sidebar stops working when we call window.DocsifyCopyCodePlugin.init(),

### ISSUE THREAD

**Issue by mohsens22** [QingWei-Li/docsify#462]
Hello.
According to #459 and #461 sidebar stops working when we call window.DocsifyCopyCodePlugin.init(), in code.
take a look at [this repository](https://github.com/Geeksltd/Programming.Tips) and place this code(as it was before). It stops sidebar.

**Comment by adrianbj** [comment]
Just wanted to say that I can confirm this and to mention that it also breaks the coverpage. As far as I know, everything was working a day or two ago.

You can see the effects on this site also: https://adrianbj.github.io/TracyDebugger/

**Comment by adrianbj** [comment]
Instructions on how to fix here: https://github.com/jperasmus/docsify-copy-code/issues/7

Making these changes has everything working again for me.

**Comment by mohsens22** [comment]
So the bug is from `docsify-copy-code`?
Even if this solution work (it does) this issue should be open till `docsify-copy-code` is fixed.

**Comment by jperasmus** [comment]
This is an issue with `docsify-copy-code` because we've updated the plugin to a new major version, meaning it has breaking changes.

You no longer need to manually add the stylesheet or call `window.DocsifyCopyCodePlugin.init()` to initialize the plugin, it will initialize automatically. These changes were made in preparation for better theming support going forward. The changes also include better support for older browsers.

Apologies for the inconvenience.

In short, for others experiencing the same, you can remove all previous code needed to add the plugin and replace it with the following `script` tag below the docsify `script` tag:

```
<script src="https://unpkg.com/docsify-copy-code"></script>
```

**Comment by jhildenbiddle** [comment]
FYI...

The new URL allows unpkg.com to redirect requests to the appropriate file. This is better than linking directly to a developer may change file names or locations. Using the redirect makes this a non-issue.

As for the trailing slash...

- Without slash = redirect to script file: https://unpkg.com/docsify-copy-code
- With slash = list files in a package: https://unpkg.com/docsify-copy-code/

Appending a `A[number]` after the package name let's you specify versions:

- v1.xx redirect = https://unpkg.com/docsify-copy-code@1
- v1.xx files = https://unpkg.com/docsify-copy-code@1/
- v1.xx redirect = https://unpkg.com/docsify-copy-code@2
- v1.xx files = https://unpkg.com/docsify-copy-code@2/

😄
